trk/src/Screens/TabScreens/Record/Backend/logic.js

```diff
@@ -125,11 +163,14 @@ export const useHomeScreenLogic = (props) => {
                 </>
             );
         } else {
+
+            const messageComponent = selectedMessage.split(/(?<=[.,!?  ])\s/).map((sentence, index) => (
+                <Text key={index} style={index === 0 ? styles.tapText : styles.sentence}>{sentence}</Text>
+            ));
+
             return (
                 <View style={{ flex: 1 }}>
-                    <Text style={styles.tapText}>Completed a climb?</Text>
-                    <Text style={styles.celebration}>🎉🎉🎉</Text>
-                    <Text style={styles.instructions}>Tap below to save your achievement</Text>
+                    {messageComponent}
                     <TouchableOpacity style={styles.button} onPress={identifyClimb}>
```

Comment:
this message component (containing one or more <Text></Text> components) is rendered

Reply:
Hmm, was just wondering how it would render. Looks neater with the celebration emojis. How do you think we should format it?

Reply:
True that celebration emojis bring a lot of positive energy. Feedback noted!

Reply:
Yeah we can definitely implement emojis, i removed them to make space for the climbItem that gets added upon successful read